app/tests/spec/lib/app-start.js

```diff
@@ -82,33 +92,26 @@ function (chai, sinon, Raven, AppStart, Session, Constants, p, Url, OAuthErrors,
           });
       });
 
-      it('redirects to /cookies_disabled if localStorage is disabled', function () {
-        appStart.useConfig({
-          localStorageEnabled: false,
-          i18n: {
-            supportedLanguages: ['en'],
-            defaultLang: 'en'
-          }
-        });
+      it('does not redirect', function () {
+        sinon.spy(routerMock, 'navigate');
+        return appStart.startApp()
+          .then(function () {
+            assert.isFalse(routerMock.navigate.called);
+          });
+      });
 
+      it('redirects to /cookies_disabled if local storage is disabled', function () {
+        storageAbility.isLocalStorageEnabled = false;
```

Comment:
This is where sinon is handy, you could do something like:
```js
sinon.stub(storage, 'isLocalStorageEnabled', function () {
  return false;
});
```

We started moving away from creating new configurable mock objects once we realized we could do that.